### PR TITLE
(DRAFT) RC24 Dolphin guide update

### PR DESCRIPTION
**Pull request by Naim2000**
https://dolphin-emu.org/download/dev/c0476fdac30f764ddce3cb3381271fffd14269d9/ :D

### CODE REVIEW

File: _pages/en_US/riiconnect24-dolphin.md [hunk 1]
```diff
@@ -37,7 +36,8 @@ DO NOT INSTALL RIICONNECT24 ON A WII MINI! It will not work and it will brick th
 If you have Dolphin already installed, skip to Section II
 {: .notice--info}
 
-1. Download the latest Dolphin beta version, but **not a stable version as these are very outdated!**
+1. Download the latest Beta version of Dolphin.
+Do not be misled by the title "Stable". These versions are very old and are not very stable. {: .notice--info}
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:40]:
```suggestion
Do not be misled by the title "Stable". These versions are very old and are not very stable. 
{: .notice--info}
```
needs a newline to render correctly

File: _pages/en_US/riiconnect24-dolphin.md [hunk 2]
```diff
@@ -48,63 +48,44 @@ If you have a homebrewed Wii console, you can use a [BootMii NAND dump](bootmii)
...
+12. After the patching process has finished, press Enter to return to the main menu and `4` to exit the patcher.
+13. In Dolphin, press `Tools` and then `Install WAD`. Go to the directory the patcher is located, then find `rc24-data/Copy-To-SD/WAD` and install
+each of the `.wad` files located here.
 </div>
 
-You're all done! Unfortunately, the Nintendo Channel and Wii Mail don't work in Dolphin yet.
+You're all done! Unfortunately, Wii Mail doesn't work in Dolphin yet.
 {: .notice--info}
 
+If a channel says WiiConnect24 features cannot be used, try reset the channel. They don't seem to work the first time. {: .notice--info}
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:88]:
```suggestion
If a channel says WiiConnect24 features cannot be used, try to reset the channel. They don't seem to work the first time. 
{: .notice--info}
```

**Comment by Naim2000** [comment]
Ok i don't think i use draft pull requests like this

**Review by osfanbuff63 — COMMENTED** [review]
#289 exists, but I think this is probably the better version to go for at least for now, since RiiConnect24 isn't built into Dolphin (as in, you still need to run the patcher) *yet*.
If you could get new Unix screenshots that would be great, otherwise that's fine too.

**Comment by Naim2000** [comment]
Screenshot from 2022-11-06 14-11-08
Screenshot from 2022-11-06 14-11-15
Screenshot from 2022-11-06 14-11-26
 I took a few. Do i need more

**Review by osfanbuff63 — APPROVED** [review]
Once we get a new beta release of Dolphin, - which I think should be soon given it's been 2 months - probably worth adding that back to recommendations, for now it's fine though as the support doesnt exist yet in beta lol

**Comment by osfanbuff63** [comment]
> Screenshot from 2022-11-06 14-11-08 Screenshot from 2022-11-06 14-11-15 Screenshot from 2022-11-06 14-11-26 I took a few. Do i need more

Looks fine

**Comment by Naim2000** [comment]
> Once we get a new beta release of Dolphin, - which I think should be soon given it's been 2 months - probably worth adding that back to recommendations, for now it's fine though as the support doesnt exist yet in beta lol
Yeah this is why I didn't change step 1

File: _pages/en_US/str2hax.md [hunk 3]
```diff
@@ -53,5 +53,7 @@ This exploit requires you to set your DNS in order to connect to a server that c
 
 If the HackMii Installer doesn't load and it instead freezes (you can't move your cursor) or it doesn't take 1-2 minutes to trigger, please restart your Wii and retry the exploit.
 
+If you have installed a mod like CTGP Revolution or Project+, str2hax may load that instead. If it does, restart your Wii and try again without your SD card inserted.
```
Comment (Peter0x44) [inline _pages/en_US/str2hax.md:56]: I don't think you meant to include this... perhaps you need to rebase?

Comment (Naim2000) [reply _pages/en_US/str2hax.md:56]: Yeah I suck at git
how exactly do i rebase now

**Comment by Naim2000** [comment]
image
Aaaahhhh git

**Comment by noahpistilli** [comment]
Please squash your changes

**Comment by larsenv** [comment]
Sorry, I already made the changes myself. I'm closing this.

**Pull request closed without merging**